Dockerfile.rhel7

```diff
@@ -0,0 +1,17 @@
+FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.10 AS builder
+
+ADD . /usr/src/ib-sriov-cni
+
+WORKDIR /usr/src/ib-sriov-cni
+RUN make clean && \
+    make build
+
+FROM registry.svc.ci.openshift.org/ocp/4.0:base
+COPY --from=builder /usr/src/ib-sriov-cni/build/ib-sriov /usr/bin/
+WORKDIR /
+
+LABEL io.k8s.display-name="InfiniBand SR-IOV CNI"
+
+ADD ./images/entrypoint.sh /
```

Comment:
why node use COPY see https://nickjanetakis.com/blog/docker-tip-2-the-difference-between-copy-and-add-in-a-dockerile

Reply:
No particular reason, I just copied from existing dockerfiles such as Dockerfile and Dockerfile.ppc64le.
I think both COPY and ADD shall work, but COPY as mentioned in above article is more explicit. I can make the change to all dockerfiles if that looks good.